tests/core/netplan-cfg/task.yaml

```
@@ -1,5 +1,11 @@
 summary: Ensure that netplan config integration works
 
+details: |
+    Check the netplan configuration is correct in Ubuntu Core 20+. Verify
+    that is is possible to add a br54 interface via netplan, including the
+    configuration for dhcp4 and dhcp6. Verify the unset for the whole br54
+    subtree works
+
 # TODO: enable for all ubuntu core versions once netplan.io got SRUed
 #       into the older versions of ubuntu
 systems:
```

Small nitpick, The test currently doesn't run for uc24, either the details section reflects this or the systems section is updated.
```suggestion
systems:
    - ubuntu-core-2*
```